src/Dialect/ONNX/IndexExpr.hpp

```diff
@@ -399,10 +402,15 @@ class IndexExpr {
 public:
   // Default and shallow copy constructors. Index expressions are usually built
   // using the subclasses listed as friend above.
-  IndexExpr() : indexExprObj(nullptr) {} // Undefined index expression.
+  IndexExpr() = default;
```

Comment:
I really need `indexExprObj` to be set to nullptr. Is that the default?

Reply:
Yes it is because at line 557  I initialized it explicitly to `nullptr`.

Reply:
Oh, so you prefer putting the default value in the declaration than the constructor? Is that because when it is set on the declaration, it then apply by default to all of the constructors? I appreciate your input as I would like to understand the suggested changes... which are for the best if that is what is actually happening.